README.md

```diff
@@ -1,43 +1,141 @@
-# Official Cognite Data Fusion project templates
+# Official Cognite Data Fusion Project configuration templates
 
-The CDF Toolkit is a command-line interface (`cdf-tk`) used for configuring and administrating Cognite Data
-Fusion (CDF) projects.
-It supports three different modes of operation:
+> Configure Cognite Data Fusion quickly, consistently, traceably, and repeatably.
+ 
+> [!WARNING]
+> The templates and the `cdf-tk` tool are currently in **ALPHA**. The scope in alpha is on Asset
+Performance Management focused on Infield (Digital Operator Rounds). The templates and tooling
+will be continously improved throughout moving towards beta and general availability.
 
-1. As an **interactive command-line tool** used alongside the Cognite Data Fusion web application to retrieve and
-   push configuration of the different Cognite Data Fusion services like data sets, data models, transformations,
-   and more. This mode also supports configuration of new Cognite Data Fusion projects to quickly get started.
-2. As tool to support the **project life-cyle by scripting and automating** configuration and management of Cognite Data
-   Fusion projects where CDF configurations are kept as yaml-files that can be checked into version
-   control. This mode also supports DevOps workflows with development, staging, and production projects.
-3. As a **tool to deploy official Cognite project templates** to your Cognite Data Fusion project. The tool comes
-   bundled with templates useful for getting started with Cognite Data Fusion, as well as for specific use cases
-   delivered by Cognite or its partners. You can also create your own templates and share them.
 
-## ALPHA NOTE
+## Getting started
 
-The templates and the `cdf-tk` tool are currently in ALPHA. The scope in alpha is on Asset
-Performance Management focused on Infield (Digital Operator Rounds). The templates and tooling
-will be continously improved throughout moving towards beta and general availability.
+This are the steps to get started with Cognite Data Fusion configuration:
```

Comment:
When you have a list of steps first, and then you detail out those below, you will always loose a certain percentage who will not read on, but still be confused. Thus, all step descriptions should be actionable for those who do not want to read on.